### PR TITLE
New version: Vendicated.Vencord.Canary version 1.4.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135799)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 638.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Actual hash:   B84EE234DE8091D9092D0BA591093315234D0DC917C444131750959F09610246
 Expected hash: f33967cdc61d0f2e720c335880fb3350dfeaadb68e8a9fdd16787e94cd22c8de

(Automated response - build 652.)

File: manifests/v/Vendicated/Vencord/Canary/1.4.0/Vendicated.Vencord.Canary.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  Custom: -branch canary
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Discord.Discord.Canary
+ReleaseDate: 2024-01-23
+RequireExplicitUpgrade: true
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/Vencord/Installer/releases/download/v1.4.0/VencordInstallerCli.exe
+  InstallerSha256: F33967CDC61D0F2E720C335880FB3350DFEAADB68E8A9FDD16787E94CD22C8DE
```
Comment (stephengillie) [inline manifests/v/Vendicated/Vencord/Canary/1.4.0/Vendicated.Vencord.Canary.installer.yaml:20]:
```suggestion
  InstallerSha256: B84EE234DE8091D9092D0BA591093315234D0DC917C444131750959F09610246
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Actual hash:   B84EE234DE8091D9092D0BA591093315234D0DC917C444131750959F09610246
 Expected hash: f33967cdc61d0f2e720c335880fb3350dfeaadb68e8a9fdd16787e94cd22c8de

(Automated response - build 655.)

**Pull request closed without merging**